spec/lib/email_alert_signup_api_spec.rb

```diff
@@ -20,22 +21,27 @@
   let(:facets) { [] }
   let(:subscriber_list_title) { "Subscriber list title" }
   let(:finder_format) {}
+  let(:default_frequency) { nil }
+
+  def init_simple_email_alert_api(subscription_url)
+    email_alert_api_has_subscriber_list(
+      "tags" => {},
+      "subscription_url" => subscription_url
+    )
+
+    expect(Services.email_alert_api).to receive(:find_or_create_subscriber_list).with(
+      "tags" => {},
+      "title" => subscriber_list_title,
+    ).and_call_original
+  end
 
   describe "default_attributes" do
     context "no default_attributes or attributes" do
       describe "#signup_url" do
         let(:subscription_url) { "http://gov.uk/email" }
 
         it "returns the url email-alert-api gives back" do
-          email_alert_api_has_subscriber_list(
-            "tags" => {},
-            "subscription_url" => subscription_url
-          )
-
-          expect(Services.email_alert_api).to receive(:find_or_create_subscriber_list).with(
-            "tags" => {},
-            "title" => subscriber_list_title,
-          ).and_call_original
+          init_simple_email_alert_api subscription_url
```

Comment:
I feel this should be written with parentheses, to make it consistent with the rest of the code in this repo
```suggestion
          init_simple_email_alert_api(subscription_url)
```

Reply:
I thought it was Ruby convention to omit parentheses - but I've just done some Googling and perhaps that isn't the case! I agree adding parentheses is clearer 👍

This is the sort of thing that really should be picked up by [govuk-lint](https://github.com/alphagov/govuk-lint) - I'd like to raise a PR there to avoid these conversations reoccurring!

I'll go back and capitalise the commit messages too.

Reply:
Hmmm adding it to govuk-lint is easier said than done as it breaks things that are in the DSL (e.g. `expect(foo) to(bar)`) and there's [no way of telling what's in the DSL](https://github.com/rubocop-hq/rubocop/issues/714).

Have done it in the repo now though!